src/main/java/jag/FindCommand.java

```diff
@@ -0,0 +1,47 @@
+package jag;
+
+/**
+ * A FindCommand Class that extends the Command Class. It is responsible for
+ * finding a given task in a taskList upon calling the command as a user input
+ */
+public class FindCommand extends Command {
+
+    /**
+     * Call upon the Ui instance to get the description that is
+     * to be found,
+     *
+     * @param tasks Tasklist instance to be used to find any task that
```

Comment:
Small nit here, but could it be more readable if you wrote your descriptions of the params more concisely?